mycroft/skills/intent_service.py

```diff
@@ -166,6 +187,7 @@ def do_converse(self, utterances, skill_id, lang, message):
             lang (str): current language
             message (Message): message containing interaction info.
         """
+        LOG.debug("do_converse()-utterances:%s, active_skills=%s" % (utterances, self.active_skills))
```

Comment:
%s notation for string variable resolution is old-school.  use an f-string or {}.format instead.  this happens in a few places in this PR